### PR TITLE
feat(rutorrent): sets `$localHostedMode = true`

### PR DESCRIPTION
**Pull request by blueberryapple**
## Description
Sets `$localHostedMode = true` since this docker image always has rTorrent and ruTorrent on the same machine.

## Context
- https://github.com/Novik/ruTorrent/commit/aeb9f62e3f3a0152a9f0bbc8a2007b206f231098
- https://github.com/Novik/ruTorrent/issues/2427#issuecomment-1407302954
- [release note](https://github.com/Novik/ruTorrent/releases#:~:text=Add%20localHostedMode%20feature%2C%20to%20improve%20loading%20times%20under%20high%20load)

### CODE REVIEW

**Comment by stickz** [comment]
You can explicitly set this value to true because ruTorrent is hosted on the same machine as rTorrent in this docker image. You will need to upgrade from ruTorrent **v4.0** to **v4.0.1** to use the feature.

Thoughts on upgrading to ruTorrent **v4.0.1** @crazy-max ? More polished version with a few crucial fixes.

**Comment by crazy-max** [comment]
> You can explicitly set this value to true because ruTorrent is hosted on the same machine as rTorrent in this docker image. You will need to upgrade from ruTorrent **v4.0** to **v4.0.1** to use this feature.

Yes should default to true agree.

> Thoughts on upgrading to ruTorrent **v4.0.1** @crazy-max ? More polished version with a few crucial fixes.

Already done, see https://github.com/crazy-max/docker-rtorrent-rutorrent/releases/tag/4.0.1-0.9.8-0.13.8-r0

File: rootfs/etc/cont-init.d/03-config.sh [hunk 1]
```diff
@@ -220,6 +221,10 @@ cat > /var/www/rutorrent/conf/config.php <<EOL
 \$log_file = '${RU_LOG_FILE}';
 \$do_diagnostic = ${RU_DO_DIAGNOSTIC};
 
+// If true, skip checking for remote exes.
+// Set to true if rTorrent is hosted on the SAME machine as ruTorrent.
+\$localHostedMode = ${RU_LOCAL_HOSTED_MODE};
```
Comment (crazy-max) [inline rootfs/etc/cont-init.d/03-config.sh:226]: Just set the same as https://github.com/Novik/ruTorrent/blob/21b6a27ea1a97b56720d7e05c3eb378103cef77a/conf/config.php#L62 and enforce to true.

```suggestion
\$localHostedMode = true; 		// Set to true if rTorrent is hosted on the SAME machine as ruTorrent
```

No need to have an env var for this.

**Review by crazy-max — CHANGES_REQUESTED** [review]
Don't think we need `RU_LOCAL_HOSTED_MODE` at all. Enforcing to true should be enough.

**Review by crazy-max — APPROVED** [review]

**Comment by BloodyIron** [comment]
Yay! Thanks for the work on this @stickz and @crazy-max ! \o/ This (and related hotfix) really helped me with load times by a lot! :D

**Pull request merged**